fix for 3.10

collections has no attribute Mapping

`ipyvolume==0.6.0a10;python_version>='3.10' `
this fixes that error however this is not the only thing that is required to fix for `3.10`:

`ipysheet 0.5.0` is incompatible with `ipywigets 8.x` and looks the issue that @oruebel opened has not been resolved:
https://github.com/QuantStack/ipysheet/issues/231
https://github.com/NeurodataWithoutBorders/nwbwidgets/issues/220
I figured out how to resolve that with using @widgets.register instead of @widgets.register('ipysheet.Sheet'), but the tests are still not passing. Now I have something else:

```
        backend_mod = importlib.import_module(
            cbook._backend_module_name(newbackend))
>       canvas_class = backend_mod.FigureCanvas
E       AttributeError: module 'backend_interagg' has no attribute 'FigureCanvas'
```

/Users/weian/opt/anaconda3/envs/nwbwidgets/lib/python3.10/site-packages/matplotlib/pyplot.py:266: AttributeError

It looks like there is still a package incompatibility issue yet to be solved, but not sure how to proceed.
I have `matplotlib-3.6.0`

Hi, sorry to come here uninvited 

```
         backend_mod = importlib.import_module(
            cbook._backend_module_name(newbackend))
>       canvas_class = backend_mod.FigureCanvas
E       AttributeError: module 'backend_interagg' has no attribute 'FigureCanvas'
/Users/weian/opt/anaconda3/envs/nwbwidgets/lib/python3.10/site-packages/matplotlib/pyplot.py:266: AttributeError
```

I may be wrong but I believe it's unlikely ipysheet is the culprit for this error, as ipysheet is not using Matplotlib.

> Perhaps we could replace that with something else

If you just want to draw a table, I can suggest looking at [ipydatagrid](https://github.com/bloomberg/ipydatagrid) which is more performant than ipysheet.

Also ipysheet depends on an old unmaintained version of Handsontable (Handsontable is not open-source anymore), so it won't get updated so often now. This issue was one of the reasons to start creating ipydatagrid.

--- a/nwbwidgets/base.py
+++ b/nwbwidgets/base.py
@@ -5,7 +5,7 @@ from typing import Union
 import pandas as pd
 import matplotlib.pyplot as plt
 
-import ipysheet
+from ipydatagrid import DataGrid
 from IPython import display
 from ipywidgets import widgets
 from ipywidgets.widgets.interaction import show_inline_matplotlib_plots
@@ -13,6 +13,7 @@ from ipywidgets.widgets.interaction import show_inline_matplotlib_plots
 import h5py
 from pynwb import ProcessingModule
 from pynwb.core import NWBDataInterface, MultiContainerInterface
+from pynwb.base import DynamicTable
 
 from nwbwidgets import view
 
@@ -32,11 +33,14 @@ def show_fields(node, **kwargs) -> widgets.Widget:
     return vbox
 
 
-def render_dataframe(df):
-    out1 = widgets.Output()
-    with out1:
-        display.display(df.to_dataframe())
-    return out1
+def render_dataframe(dynamic_table: DynamicTable):
+    try:
+        return DataGrid(dynamic_table.to_dataframe())
+    except:
+        out1 = widgets.Output()
+        with out1:
+            display.display(dynamic_table.to_dataframe())
+        return out1
 
 
 def show_neurodata_base(
@@ -325,22 +329,9 @@ def show_dset(dset: h5py.Dataset, **kwargs):
 
 
 def dataset_to_sheet(dset: h5py.Dataset):
-    if dset.ndim == 1:
-        nrows = len(dset)
-
-        sheet = ipysheet.easy.sheet(rows=nrows, columns=1, column_headers=False)
-        for row in range(nrows):
-            ipysheet.easy.cell(row, 0, dset[row], read_only=True)
-    elif dset.ndim == 2:
-        nrows, ncols = dset.shape
-
-        sheet = ipysheet.easy.sheet(rows=nrows, columns=ncols, column_headers=False)
-        for row, col in zip(range(nrows), range(ncols)):
-            ipysheet.easy.cell(row, col, dset[row, col], read_only=True)
-    else:
-        # do not know how to render datasets that have 3 or more dimensions
+    if dset.ndim >= 2:
         return widgets.HTML(print(dset))
-    return sheet
+    return DataGrid(pd.DataFrame(dset[:]))
 
 
 def show_dict(in_dict) -> widgets.Widget:

--- a/nwbwidgets/version.py
+++ b/nwbwidgets/version.py
@@ -1,1 +1,1 @@
-version = "0.8.1"
+version = "0.9.0"

--- a/requirements.txt
+++ b/requirements.txt
@@ -3,7 +3,7 @@ matplotlib
 scikit-image
 pynwb
 ipympl
-ipysheet
+ipydatagrid
 ipyvolume>=0.6.0a10
 ipywidgets
 plotly

